indico/modules/events/registration/models/items.py

```diff
@@ -51,6 +51,12 @@ class RegistrationFormItemType(int, IndicoEnum):
 
 
 class PersonalDataType(int, IndicoEnum):
+    """Description of the personal data items that exists on every registration form
+
+    We are not using an TitledIntEnum since one of the instances is named "title".
```

Comment:
"..a TitledIntEnum..", and I would put this in a comment instead of the docstring.